charts/portworx/templates/storage-cluster.yaml

```diff
@@ -84,13 +84,21 @@ spec:
     kvdbDevice: {{ $kvdbDevice }}
     {{- end }}
 
+  {{- if .AKSInstall }}
+  cloudStorage:
```

Comment:
One thing we need to consider when "$drives" are specified is that it could be a cloud storage spec. We need some logic in the helm chart to decide whether to populate the `spec.storage` section `spec.cloudStorage` section.

Reply:
Yup. We do something similar in the px-installer/spec-generator.
https://github.com/portworx/px-installer/blob/a021d9c6739490d43a1e40e406f96c50faac30ae/px-spec-websvc/operator.go#L85
https://github.com/portworx/px-installer/blob/master/px-spec-websvc/templates/template_storage_cluster.gtpl#L72

Reply:
Feel free to do the change in another PR if you want.